### PR TITLE
Open binary

### PR DESCRIPTION
**Pull request by chr1sgrant**

### CODE REVIEW

**Comment by chr1sgrant** [comment]
includes #277 and #280

File: microArch/factory/make_plate_library.go [hunk 1]
```diff
@@ -330,11 +330,11 @@ func makePlateLibrary() map[string]*wtype.LHPlate {
 	wellyoffset = 18.0 //centre of well to centre of neighbouring well in y direction
 	xstart = 5.0       // distance from top left si z9fdJwde of plate to first well
 	ystart = 5.0       // distance from top left side of plate to first well
-	zstart = 7.0       // offset of bottom of deck to bottom of well
+	zstart = 7.0 - 1.0 // offset of bottom of deck to bottom of well
```
Comment (ddn0) [inline microArch/factory/make_plate_library.go]: separate plate library changes to another PR

Comment (chr1sgrant) [inline microArch/factory/make_plate_library.go]: These changes are from PR #277 and already in master anyway
